compose/client/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM node:10.16.0
```

Comment:
@wlonk Would it be possible to pull this from `package.json` instead of duplicating it here?

Reply:
I'm not sure how we might. We could set the value at `Dockerfile`/`package.json` creation time, but I think getting the value at runtime is gonna be tricky.

If we set up a makefile, it could have a "copy the value in to the Dockerfile" step before build?

Reply:
Not a big deal, just curious 👍